cedar-policy-core/src/est/expr.rs

```diff
@@ -1076,7 +1076,7 @@ fn interpret_primary(p: cst::Primary) -> Result<Either<ast::Name, Expr>, ParseEr
                 }
             }
             Some(cst::Ref::Ref { .. }) => {
-                Err(ParseError::ToAST("entity-lookup syntax is not supported".to_string()).into())
+                Err(ParseError::ToAST("entity-lookup syntax is not supported: please use `Namespace::EntityType::EntityID` or similar".to_string()).into())
```

Comment:
should this be `Namespace::EntityType::"EntityID"` to remind people quotes are required?